torch/onnx/symbolic_opset11.py

```diff
@@ -111,6 +123,8 @@ def __interpolate(g, input, size, scale_factor, mode, align_corners):
                     mode_s=mode,  # nearest, linear, or cubic
                     nearest_mode_s="floor")
     else:  # if not sym_help._is_none(scales)
+        if not input.isCompleteTensor():
```

Comment:
Why is the complete tensor requirement a must have for the scales route?

Reply:
input.type().dim() used line 128. just relaxed this to verify if rank is available.